src/lib.rs

```diff
@@ -448,20 +455,49 @@ where
             if next_char == Some('{') {
                 match input_str.find('}') {
                     Some(closing_brace_idx) => {
-                        let var_name = &input_str[2..closing_brace_idx];
-                        match try_lookup!(var_name, context(var_name)) {
-                            Some(var_value) => {
+                        let mut default_value = None;
+
+                        // Search for the default split
+                        let env_name_end_idx = match input_str.find(":-") {
+                            Some(default_split_idx) => {
+                                default_value = Some(&input_str[default_split_idx + 2..closing_brace_idx]);
+                                default_split_idx + 1
```

Comment:
Why +1? If `default_split_idx` points to `:` in `:-`, then +1 would mean `var_name` will include the trailing `:`, or do I understand it incorrectly?

Reply:
You are right, I missed that when I refactored part of the code